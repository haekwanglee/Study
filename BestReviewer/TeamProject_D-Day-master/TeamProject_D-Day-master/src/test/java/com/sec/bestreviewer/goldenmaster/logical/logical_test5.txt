SCH,-p,-l, ,phoneNum,1111,-a,-d, ,birthday,01
MOD,-p,-l, ,phoneNum,1111,-a,-d, ,birthday,01,certi,PRO
SCH,-p,-l, ,phoneNum,1118,-o,-d, ,birthday,09
SCH, ,-l, ,phoneNum,1118,-o,-d, ,birthday,09
MOD,-p,-l, ,phoneNum,1118,-o,-d, ,birthday,09,certi,PRO
SCH, ,-l, ,phoneNum,1118,-o,-d, ,birthday,09