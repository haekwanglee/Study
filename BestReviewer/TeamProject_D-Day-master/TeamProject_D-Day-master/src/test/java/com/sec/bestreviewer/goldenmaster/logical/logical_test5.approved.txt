SCH,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
SCH,95000002,BBBBB BBB,CL2,010-2222-1111,19980201,ADV
SCH,00000001,DICTIONARY A,CL2,010-3333-1111,19980301,ADV
SCH,20000003,DICTIONARY ACC,CL2,010-4444-1111,19980801,ADV
SCH,33000001,CERTIADV BABY,CL2,010-5555-1111,20000701,ADV
MOD,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
MOD,95000002,BBBBB BBB,CL2,010-2222-1111,19980201,ADV
MOD,00000001,DICTIONARY A,CL2,010-3333-1111,19980301,ADV
MOD,20000003,DICTIONARY ACC,CL2,010-4444-1111,19980801,ADV
MOD,33000001,CERTIADV BABY,CL2,010-5555-1111,20000701,ADV
SCH,95000000,BBBBB B,CL2,010-1111-1118,19980108,ADV
SCH,95000001,BBBBB BB,CL2,010-1111-1119,19980109,ADV
SCH,99999999,LASTTWENTYCENTURY BABY,CL2,010-2222-1118,19980208,ADV
SCH,00000000,MILLENNIAR BABY,CL2,010-2222-1119,19980209,ADV
SCH,10000003,DICTIONARY ACB,CL2,010-3333-1118,19980308,ADV
SCH,8
MOD,95000000,BBBBB B,CL2,010-1111-1118,19980108,ADV
MOD,95000001,BBBBB BB,CL2,010-1111-1119,19980109,ADV
MOD,99999999,LASTTWENTYCENTURY BABY,CL2,010-2222-1118,19980208,ADV
MOD,00000000,MILLENNIAR BABY,CL2,010-2222-1119,19980209,ADV
MOD,10000003,DICTIONARY ACB,CL2,010-3333-1118,19980308,ADV
SCH,8
